tasks/install-matlab/v0/test/downloadAndExecuteLinux.ts

```diff
@@ -11,7 +11,7 @@ process.env.SYSTEM_SERVERTYPE = "hosted";
 
 tr.registerMock("azure-pipelines-tool-lib/tool", {
     downloadTool(url: string) {
-        if (url !== "https://ssd.mathworks.com/supportfiles/ci/ephemeral-matlab/v0/install.sh") {
+        if (url !== "https://ssd.mathworks.com/supportfiles/ci/matlab-deps/v0/install.sh" && url !== "https://ssd.mathworks.com/supportfiles/ci/ephemeral-matlab/v0/install.sh") {
```

Comment:
```suggestion
        const expectedUrls = [
            "https://ssd.mathworks.com/supportfiles/ci/matlab-deps/v0/install.sh",
            "https://ssd.mathworks.com/supportfiles/ci/ephemeral-matlab/v0/install.sh",
        ];
        
        if (!expectedUrls.includes(url)) {
```

Perhaps something like this to remove the long if-statement?